src/lc0ctl/onnx2leela.cc

```diff
@@ -230,6 +236,77 @@ bool ValidateNetwork(const pblczero::Net& weights) {
   return true;
 }
 
+const std::vector<float> rule50weights = {
```

Comment:
Constants should have names with `k-` prefix, like `kRule50weights`.
But maybe it's better to create dynamically in function?

```cpp
std::vector<float> MakeRule50Weights() {
  std::vector res(122, 1.0f);
  res[120] = 1.0r / 00;
  return res;
}
```

Reply:
Done